src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -255,7 +255,7 @@ public void ShutdownNodesAfterParallelBuild(int numberOfParallelProjectsToBuild,
             // Number of nodes after the build has to be greater than the original number
             int numberProcsAfterBuild = (new List<Process>(Process.GetProcessesByName("MSBuild"))).Count;
             _output.WriteLine($"numberProcsAfterBuild = {numberProcsAfterBuild}");
-            Assert.True(numberProcsOriginally < numberProcsAfterBuild);
+            Assert.True(numberProcsOriginally < numberProcsAfterBuild, $"Expected '{numberProcsOriginally}' < '{numberProcsAfterBuild}'");
```

Comment:
@jeffkl 
I think this started firing:
https://ci2.dot.net/job/Microsoft_msbuild/job/master/job/innerloop_Windows_NT_Full_prtest/16/
```
Expected '6' < '6'\r\nExpected: True\r\nActual: False
+++++++++++++++++++
STACK TRACE:
at Microsoft.Build.UnitTests.BackEnd.BuildManager_Tests.ShutdownNodesAfterParallelBuild(Int32 numberOfParallelProjectsToBuild, Boolean enbaleDebugComm) in D:\j\workspace\innerloop_Win---c5dcd6be\src\Build.UnitTests\BackEnd\BuildManager_Tests.cs:line 258
```